app/mainAppWindow/index.js

```diff
@@ -511,16 +509,16 @@ function assignSelectSourceHandler() {
 	};
 }
 
-async function handleOnIncomingCallCreated() {
+async function handleOnIncomingCallCreated(e, data) {
 	if (!incomingCallCommandProcess && config.incomingCallCommand) {
```

Comment:
I wonder if it would be better to change this to always do the kill command (terminate now), if the config.incomingCallCommand is present. That way we don't end up with orphans incomingCallCommandProcess

Reply:
I am not sure if I understood you correctly.
Do you mean that the terminate function should also be called before a new execution of the command on an new incoming call?

Reply:
I put a code suggestion in another conversation, but yes.

Reply:
I put the change in myself and also changed the Version to 1.3.27. I hope that was correct.